docs/source/quickstart-docker.rst

```diff
@@ -16,6 +22,14 @@ Prerequisites
 You should have `Docker`_ installed on a Linux/Unix based system.
 
 
+
+**note:**
```

Comment:
In rst, notes are created with `.. note::`, not `**note:**`.

This duplicates the note, rather than updating it.

Reply:
Ok Sir @minrk thanks